pytest.ini

```diff
@@ -0,0 +1,14 @@
+[pytest]
+# always check coverage of janitor.
+addopts = --cov=janitor --cov-report term-missing --cov-report xml --durations=0
+markers =
+    functions: test for general functions
+    biology: tests for biology
+    chemistry: tests for chemistry
+    finance: tests for finance
+    utils: utility tests
+    engineering: tests for engineering
+    ml: tests for machine learning
+    spark_functions: tests for pyspark functions
+    xarray: tests for xarray functions
+    timeseries: tests for timeseries
```

Comment:
@samukweku I think this one is not necessary as the configurations for pytest are present in the `pyproject.toml` file.

Reply:
So I can ignore @ericmjl

Reply:
That's right, I think we should delete `pytest.ini` - could you instead combine the contents of line 3 (`addopts`) with the addopts in `pyproject.toml`?